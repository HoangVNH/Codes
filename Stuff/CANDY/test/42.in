10
70
XXVDDDVXVVXXDDVXVVDXDDVXDVDVVXXXDXVDVVDDVXXVVXXXDXVXXVDVVXDDDXXDVXDXXV
86
XXXXDDXXXXDVDXXXXDDVDXVVVVXDVDXXDDDXVVDVVDVXVDVDXDDVXDDDDDVXXXXXVVVXDDXVDDXXXXVVVXXDVV
81
XDVDXDXVXDXXVDDXVVDVVDDVXDDXDDVXDDDVXVDVXXXXDVVDXXXVVVVXVVVDXXVDXVVXXVDDVXDDDXXXD
51
XXXDXXXVDXVXXDVXXXVXVVXDDDXXXVDVDXVDDVDXDVVXDVDVVXV
97
DXDXDXXVDDVXXVDDDVDVXVVVXVDDDXVDVVXDXXXVVXDDXVDDVDXVDDVXXXVXVXXXXDDDDVDDVXXDVVDXXDXXXXXVXXDDXVDVD
79
XXXVDVDXXXDVDDVVXXDXXXVDDDDDDXDXVXXVXVXVXXDXDVVVDDXXVDXXXXVXDXVXXXXXDVXDVXDVXVX
69
XXVXDDXXXVDXXDDDXXVXDDXDDDXDDXDDXDXXDVDXXDXXXDXDDXDVXDVVXXXVXVDVXDDDX
87
XXDXVDDXXDVVXVDVXDDDVVDDVDDDXXDDVXVDXDVVDDXDDVXVXDVDXVDXXVDDVVVDDVVVXXVXXDDDDXXVXDDXDXX
95
XXXDVXDXDVDVVVVXVXVXXDXDDDDXDXDXXVXDXVDDDVXXXDVDXDDXVXXDXXXXVVXDDDDDDVVVVDDVDDVDXVXDXDVDDDXVDXD
52
XXXDVXDVDDVVXDXDVDXDXXXDXXDDXDXVVVVXVXDVXXVXVDVVDVDD
