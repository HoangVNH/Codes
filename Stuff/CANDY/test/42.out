69
85
80
50
96
78
68
85
94
51
